pkg/plugin/interfaces.go

```diff
@@ -95,3 +95,13 @@ type CreateWebhookPluginGetter interface {
 type CreateWebhook interface {
 	GenericSubcommand
 }
+
+type EditPluginGetter interface {
+	Base
+	//GetEditPlugin return the underlying Edit interface.
+	GetEditPlugin() Edit
+}
+
+type Edit interface {
```

Comment:
```suggestion
// Edit is an interface that represents a `edit` command
type Edit interface {
```

Reply:
Done

Reply:
Hi @prafull01, 

It is missing the comment.